libiocage/lib/Jail.py

```diff
@@ -60,19 +62,24 @@ class Jail:
             release (stored in `zpool/iocage/base/<RELEASE>)
     """
 
-    def __init__(self, data={}, zfs=None, host=None, logger=None, new=False):
+    def __init__(self,
+                 data: Union[str, dict]={},
```

Comment:
Why is this a union? It is never a string, but always a dict.

Reply:
because that's exactly what the pydoc comment below says

Reply:
lol, coffee. You're right, sorry!

Reply:
yeah, but, we should still figure out if we want to simplify that

as i have done with the execs that would've allowed `Union[str, List[str]]`